Recent changes in userscapce code:

http://git.kernel.org/?p=virt/kvm/qemu-kvm.git;a=commit;h=6919115a8715c34cd80baa08422d90496f11f5d7

requires more recent kernel header:

http://git.kernel.org/?p=virt/kvm/qemu-kvm.git;a=commit;h=16cc190a673f3582bca50b983026915e4811f7ab

so we simply revert for now.

Index: new/hw/device-assignment.c
===================================================================
--- new.orig/hw/device-assignment.c	2012-04-03 07:23:56.000000000 +0200
+++ new/hw/device-assignment.c	2012-04-03 07:25:07.000000000 +0200
@@ -1771,9 +1771,9 @@
     DEFINE_PROP_BIT("iommu", AssignedDevice, features,
                    ASSIGNED_DEVICE_USE_IOMMU_BIT, true),
     DEFINE_PROP_BIT("prefer_msi", AssignedDevice, features,
-                    ASSIGNED_DEVICE_PREFER_MSI_BIT, false),
+                    ASSIGNED_DEVICE_PREFER_MSI_BIT, true),
     DEFINE_PROP_BIT("share_intx", AssignedDevice, features,
-                    ASSIGNED_DEVICE_SHARE_INTX_BIT, true),
+                    ASSIGNED_DEVICE_SHARE_INTX_BIT, false),
     DEFINE_PROP_INT32("bootindex", AssignedDevice, bootindex, -1),
     DEFINE_PROP_STRING("configfd", AssignedDevice, configfd_name),
     DEFINE_PROP_END_OF_LIST(),
